collect_app/src/main/java/org/odk/collect/android/activities/FormEntryActivity.java

```diff
@@ -798,8 +799,35 @@ public void run() {
 
 
             case RequestCodes.AUDIO_CHOOSER:
+                /*
```

Comment:
Both the switch cases calling the same method. 
```
case RequestCodes.AUDIO_CHOOSER:
case RequestCodes.VIDEO_CHOOSER:
                showDialog(SAVING_DIALOG);
                Runnable runnable = new Runnable() {
                    @Override
                    public void run() {
                        saveChosenAudioVideo(intent.getData());
                    }
                };
                new Thread(runnable).start();
                break;
```

Reply:
Thanks for reminding me, and if there are any improvement can be done with my code, any suggestions would be appreciated.